spec/user_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'user'
+
+describe User do
+
+	it '' do
```

Comment:
A little concerned there are no unit tests here. What was up?

Unit tests are super-fundamental. They make sure that when you make tiny, not-quite-feature-breaking changes, you'll be flagged about it. Plus, I should be able to read off everything your app does. Finally, if you did unit tests you probably would have figured out that Users should `have_many` Peeps, which would have saved you pain in the code later on.